src/linter/ui5Types/fix/collections/jqueryFixes.ts

```diff
@@ -77,11 +77,20 @@ t.declareModule("jQuery", [
 				scope: FixScope.FirstChild,
 			})),
 		]),
+		t.namespace("resources", [
+			t.namespace("isBundle", callExpressionGeneratorFix({ // https://github.com/SAP/ui5-linter/issues/657
+				moduleName: "sap/base/i18n/ResourceBundle",
+				generator(_ctx, [moduleIdentifier], arg1) {
+					return `new ${moduleIdentifier}() instanceof ${arg1.trim()}`;
+				},
+			})),
+		],
 		// jQuery.sap.resources => ResourceBundle.create
-		t.namespace("resources", accessExpressionFix({ // https://github.com/SAP/ui5-linter/issues/521
+		accessExpressionFix({ // https://github.com/SAP/ui5-linter/issues/521
```

Comment:
I think this should be a callExpressionFix (which then requires FullExpression scope I think).

Otherwise I fear we might incorrectly replace the property access in a larger property access, e.g. `jQuery.sap.resources.foo()`

```suggestion
		callExpressionFix({ // https://github.com/SAP/ui5-linter/issues/521
			scope: FixScope.FullExpression,
```

Reply:
It's indeed a call expression. However, it should not be FullExpression scope, because the arguments are the same